### DISCUSSION TITLE
Livewire full page component won't work with laravel 9

### DISCUSSION THREAD

**Discussion by tom-bowern** [livewire/livewire#5608] [1. Help]
Following the steps here - 
https://laravel-livewire.com/docs/2.x/rendering-components#page-components

I have the code in my routes/web.php
```
use App\Http\Livewire\Admin\Access;

 Route::middleware(['user.admin'])->prefix('/admin')->group(function () {
 Route::get('/access', [Access::class]);
});

```

Im getting an error: 
```
ReflectionException
PHP 8.1.10
9.52.0
Function () does not exist
```

Presumably due to the lack of a function being passed as expected in the laravel documentation

```
"laravel/framework": "^9.11",
"laravel/jetstream": "^2.8",
"laravel/sanctum": "^2.14.1",
"laravel/socialite": "^5.5",
"laravel/tinker": "^2.7",
"livewire/livewire": "^2.5"
```

**Comment by joshhanley** [comment]
@tom-bowern please don’t open both a discussion and an issue. 

Try removing the square brackets from around the class name. 

```php
Route::get('/access', Access::class);
```

You only need the brackets if you are using Laravel’s tuple syntax with class and method together. 

Hope this helps!
